src/wallet/rpcwallet.cpp

```diff
@@ -2406,6 +2406,9 @@ static UniValue getwalletinfo(const JSONRPCRequest& request)
             "  \"paytxfee\": x.xxxx,                (numeric) the transaction fee configuration, set in " + CURRENCY_UNIT + "/kB\n"
             "  \"hdseedid\": \"<hash160>\"            (string, optional) the Hash160 of the HD seed (only present when HD is enabled)\n"
             "  \"private_keys_enabled\": true|false (boolean) false if privatekeys are disabled for this wallet (enforced watch-only wallet)\n"
+            "  \"scannning_active\": true|false     (boolean) whether the local blockchain is being scanned for wallet related transactions\n"
```

Comment:
Could make it

`scanning: None`
or
```
scanning: {
 'duration': (num),
 'progress': (num),
}
```

Reply:
Yap, I'm fine either way.

Reply:
Indeed, just a style-preference. Though, json provides structure, so might as well use it instead of returning a single huge dict

Reply:
Done.